doc/userguide/rules/lua-detection.rst

```diff
@@ -100,5 +101,28 @@ Entire script:
 
   return 0
 
+Sandbox and Available functions
+-------------------------------
+
+By default, the maximum memory and lua instruction count per execution of a detection rule will be limited.  Additionally,
+The following libraries and functions are blocked:
```

Comment:
should this now contain the allowlisted funcs instead?

Reply:
wonder if we could utilize sphinx literalinclude to keep things in sync

Reply:
Probably should list the loaded libs and allowed functions, so yes, this needs cleanup. I don't want to directly pull from the C source code, I don't think it makes for nice documentation. In theory, it should only change when we do major upgrades to the Lua version as well.